simpa/utils/processing_device.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: 2021 Division of Intelligent Medical Systems, DKFZ
+# SPDX-FileCopyrightText: 2021 Janek Groehl
+# SPDX-License-Identifier: MIT
+
+from doctest import ELLIPSIS_MARKER
+import torch
+from simpa.log import Logger
+from simpa.utils import Tags, Settings
+
+
+def get_processing_device(used_settings: Settings = None) -> torch.device:
```

Comment:
the argument is called "global_settings" in the docstring. Please rename it consistently to `global_settings`